packages/web/components/Quest/QuestFilter.tsx

```diff
@@ -34,89 +39,214 @@ export const QuestFilter: React.FC<Props> = ({
   aggregates,
   queryVariables,
   setQueryVariable,
+  roleChoices,
 }) => {
   const { user } = useUser();
   const myId = user?.id;
 
+  const limitOptions = [
+    {
+      label: '10',
+      value: '10',
```

Comment:
shouldn't the value be an int ?

Reply:
```
const values = value as ValueType[];
            const v = values[values.length - 1];
            if (v) {
              setLimit(v);
              setQueryVariable('limit', Number(v.value));
            }
```
All of the other filter variables are defined with a `type ValueType = { value: string; label: string };`, so I'd just leave it as it is.